.github/DISCUSSION_TEMPLATE/general.yml

```
@@ -0,0 +1,70 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing to the discussion! Please fill out this form as completely as possible to help us support you better.
+
+  - type: input
+    id: description
+    attributes:
+      label: What happened?
+      description: Clearly describe the problem or question you have.
+      placeholder: e.g., "When I try to launch the tool, I receive an error..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected Behavior (optional)
```

remove "(optional)" as this indicated by required stars *
```suggestion
      label: Expected Behavior
```

---

Maybe we can combine this field in the "What happened" like "Clearly describe the problem or question you have and what behavior you would have expected".